File: app/src/main/java/com/BlizzardArmory/diablo/Character/Act1.java
package com.BlizzardArmory.diablo.Character;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import org.apache.commons.lang.builder.ToStringBuilder;

public class Act1 {

    @SerializedName("completed")
    @Expose
    private Boolean completed;
    @SerializedName("completedQuests")
    @Expose
    private List<Object> completedQuests = null;

    public Boolean getCompleted() {
        return completed;
    }

    public void setCompleted(Boolean completed) {
        this.completed = completed;
    }

    public List<Object> getCompletedQuests() {
        return completedQuests;
    }

    public void setCompletedQuests(List<Object> completedQuests) {
        this.completedQuests = completedQuests;
    }

    @Override
    public String toString() {
        return new ToStringBuilder(this).append("completed", completed).append("completedQuests", completedQuests).toString();
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/character/progression/Act1.kt
package com.BlizzardArmory.model.diablo.character.progression

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Act 1.
 */
@Keep
data class Act1(

    @SerializedName("completed")
    var completed: Boolean,

    @SerializedName("completedQuests")
    var completedQuests: List<Any>
)